deps/wazuh_testing/wazuh_testing/tools/monitoring.py

```diff
@@ -373,7 +373,7 @@ def get_results(self, callback=_callback_default, accum_results=1, timeout=-1, u
                 else:
                     item = callback(self._queue.peek(position=position, block=True, timeout=self._time_step))
                     position += 1
-                if item is not None:
+                if item is not None and item is not False:
```

Comment:
```suggestion
                if item is not None and item:
```

Reply:
Done in b3d023b38d8c6abac7fb3757a859b4b3ba4574d9